extensions/wikia/AdEngine/js/lookup/adapter/appnexus.js

```diff
@@ -0,0 +1,74 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.adapter.appnexus', function () {
+	'use strict';
+
+	var bidderName = 'appnexus',
+		slots = {
+			oasis: {
+				TOP_LEADERBOARD: {
+					sizes: [
+						[728, 90],
+						[970, 250]
+					],
+					placementId: '5823300'
+				},
+				TOP_RIGHT_BOXAD: {
+					sizes: [
+						[300, 250],
+						[300, 600]
+					],
+					placementId: '5823309'
+				}
+			},
+			mercury: {
+				MOBILE_TOP_LEADERBOARD: {
+					sizes: [
+						[320, 50]
+					],
+					placementId: '5823309'
+				}
+			}
+		};
+
+	function isEnabled() {
+		// TODO: enable with wgCountries
+		return false;
+	}
+
+	function prepareAdUnit(slotName, config) {
+		return {
+			code: slotName,
+			sizes: config.sizes,
+			bids: [
+				{
+					bidder: bidderName,
+					params: {
+						placementId: config.placementId
+					}
+				}
+			]
+		};
+	}
+
+	function getAdUnits(skin) {
+		var adUnits = [],
+			slotName;
+
+		if (!slots[skin]) {
+			return [];
+		}
+
+		for (slotName in slots[skin]) {
```

Comment:
Can we use `forEach` here?
Like here: https://github.com/Wikia/app/pull/10240/files#diff-7a14ff9954fe20151dc1f68b216f2d20R77